src/mono/sample/wasm/browser-threads-minimal/Wasm.Browser.Threads.Minimal.Sample.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
this needs some additional properties to work with xharness (so the exit code can be checked), and it can work better with CI. See https://github.com/dotnet/runtime/blob/2efdd7ec8566369926a3527411b0c95f2691416c/src/mono/sample/wasm/DefaultBrowserSample.targets#L3
https://github.com/dotnet/runtime/blob/2efdd7ec8566369926a3527411b0c95f2691416c/src/mono/sample/wasm/DefaultBrowserSample.targets#L8-L9

.. and `<ExpectedExitCode>2</ExpectedExitCode>`.

Reply:
@radical I import the DefaultBrowserSample.targets - wouldn't that set the properties? So I just need to set the exit code property, or something else?

Reply:
Yeah, that should do it.